openpype/hosts/maya/api/lib_rendersettings.py

```diff
@@ -60,8 +60,7 @@ def set_default_renderer_settings(self, renderer=None):
         try:
             aov_separator = self._aov_chars[(
                 self._project_settings["maya"]
-                                      ["create"]
-                                      ["CreateRender"]
+                                      ["RenderSettings"]
                                       ["aov_separator"]
             )]
```

Comment:
Both before and after I feel the `except KeyError` code isn't the most "explicit" we have now. We are currently unable to identify whether the key error is due to the `self._aov_chars` dictionary not having the right key or whether the Project Settings didn't have the stored setting key.

I feel like both are different errors and potentially should be handled differently. No?